static/js/pages/candidates-office.js

```diff
@@ -21,10 +21,16 @@ var defaultSort = {
   house: 4
 };
 
+var officeTitleMap = {
+  president: 'president',
```

Comment:
I noticed the other two titles are capitalized but not `president`, is this intentional?